v21.2/build-a-nodejs-app-with-cockroachdb-knexjs.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Build a Simple CRUD Node.js App with CockroachDB and Knex.js
+summary: Learn how to use CockroachDB from a simple CRUD application that uses the Knex.js query builder.
+toc: true
+twitter: false
+---
+
+<div class="filters filters-big clearfix">
+    <a href="build-a-nodejs-app-with-cockroachdb.html"><button class="filter-button">Use <strong>node-postgres</strong></button></a>
+    <a href="build-a-nodejs-app-with-cockroachdb-sequelize.html"><button class="filter-button">Use <strong>Sequelize</strong></button></a>
+    <a href="build-a-nodejs-app-with-cockroachdb-knexjs.html"><button class="filter-button current">Use <strong>KnexJS</strong></button></a>
+    <a href="build-a-typescript-app-with-cockroachdb.html"><button class="filter-button">Use <strong>TypeORM</strong></button></a>
+</div>
+
+This tutorial shows you how build a simple Node.js application with CockroachDB and [Knex.js](https://knexjs.org/).
+
+## Step 1. Start CockroachDB
+
+<div class="filters clearfix">
+  <button class="filter-button page-level" data-scope="cockroachcloud">Use {{ site.data.products.serverless }}</button>
+  <button class="filter-button page-level" data-scope="local">Use a Local Cluster</button>
+</div>
+
+<section class="filter-content" markdown="1" data-scope="cockroachcloud">
+
+### Create a free cluster
+
+{% include cockroachcloud/quickstart/create-a-free-cluster.md %}
+
+### Set up your cluster connection
+
+1. Navigate to the cluster's **SQL Users** page, and create a new user, with a new password.
```

Comment:
I just tested this and it looks like a user is already created by default when spinning up a new serverless cluster. Do we need a second user on top of this?

Reply:
(*I replaced the `### Set up your cluster connection` section with a new include. See my general comment about that new include file.*)

I don't think that we do, tbh. Removed this step in `_includes/v21.2/app/sample-setup.md`, `_includes/v21.1/app/sample-setup.md`, and `_includes/v20.2/app/sample-setup.md`.